### PR TITLE
Add video error message

### PR DESCRIPTION
**Pull request by awolant**
#### Why we need this PR?
As a follow up for #1041  

#### What happend in this PR?
 It adds meaningful error message for variable frame rate videos and docs update.

### CODE REVIEW

**Comment by awolant** [comment]
!build

File: dali/pipeline/operators/reader/loader/video_loader.cc [hunk 1]
```diff
@@ -215,6 +215,7 @@ OpenFile& VideoLoader::get_or_open_file(const std::string &filename) {
     // 1/frame_rate is duration of each frame (or time base of frame_num)
     file.frame_base_ = AVRational{stream->avg_frame_rate.den,
                                   stream->avg_frame_rate.num};
+    DALI_ENFORCE(file.frame_base_.num == 1, "Variable frame rate videos are unsupported.");
```
Comment (JanuszL) [inline dali/pipeline/operators/reader/loader/video_loader.cc]: Please add `filename` to error message (in case user provides more than one to the reader).

Comment (awolant) [reply dali/pipeline/operators/reader/loader/video_loader.cc]: Done

**Comment by awolant** [comment]
Added the test for this. Fixed the typo in the PR name.

**Comment by awolant** [comment]
!build

File: dali/pipeline/operators/reader/loader/video_loader.cc [hunk 2]
```diff
@@ -215,6 +215,9 @@ OpenFile& VideoLoader::get_or_open_file(const std::string &filename) {
     // 1/frame_rate is duration of each frame (or time base of frame_num)
     file.frame_base_ = AVRational{stream->avg_frame_rate.den,
                                   stream->avg_frame_rate.num};
+    DALI_ENFORCE(
+      file.frame_base_.num == 1,
```
Comment (klecki) [inline dali/pipeline/operators/reader/loader/video_loader.cc]: I would maybe comment that this is only heuristic based on what you found out.

Comment (awolant) [reply dali/pipeline/operators/reader/loader/video_loader.cc]: Done

**Comment by awolant** [comment]
!build

**Comment by awolant** [comment]
!build

**Comment by awolant** [comment]
!build

**Pull request merged**